tests/subproc_outputs/lldpctl.json

```diff
@@ -92,11 +92,13 @@
               ]
             }
           },
-          "vlan": {
-            "vlan-id": "146",
-            "pvid": true,
-            "value": "ntp"
-          },
+          "vlan": [
```

Comment:
>vlan [](start = 11, length = 4)

If there is only one vlan entry, "vlan" value is a dictionary. If there are multiple vlan entries, "vlan" value is a list. Better to test both cases.